test/gtest/uct/ib/test_ib.cc

```diff
@@ -108,6 +125,7 @@ UCS_TEST_P(test_uct_ib_addr, address_pack) {
     test_address_pack(UCT_IB_LINK_LOCAL_PREFIX);
     test_address_pack(UCT_IB_SITE_LOCAL_PREFIX | htobe64(0x7200));
     test_address_pack(0xdeadfeedbeefa880ul);
+    test_address_pack(0l);
```

Comment:
why 01?

Reply:
are you what about?
not 01, there is 0l (0L, zero-long) :)

Reply:
ok